muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/records/index.ts

```diff
@@ -8,13 +8,13 @@ import {
   TranscriptOfRecordLocationType,
   CurrentStudentUserAndWorkspaceStatusType,
   CurrentRecordType,
-  RecordWorkspaceActivityInfo,
   RecordWorkspaceActivityByLine,
```

Comment:
Out of scope, but this needs to be addressed somewhere. These are not lines, they are studyPrograms.